src/main/java/com/example/MainTask1.java

```diff
@@ -1,14 +1,15 @@
+//1.Приветствовать любого пользователя при вводе его имени через командную строку.
+
 package com.example;
+
 import java.util.Scanner;
 
 public class MainTask1 {
-    static class Test {
-        public static void main(String args[]){
-            Scanner in = new Scanner(System.in);
-            System.out.println("Introduce yourself: ");
-            String name = in.nextLine();
-            System.out.printf("Hello, %s", name);
-            in.close();
-        }
+    public static void main(String args[]) {
+        Scanner in = new Scanner(System.in);
```

Comment:
Please give more meaningful name to Scanner object instead of `in`. E.g. `scanner`.